roles/openshift_facts/library/openshift_facts.py

```diff
@@ -323,6 +323,24 @@ def set_fluentd_facts_if_unset(facts):
             facts['common']['use_fluentd'] = use_fluentd
     return facts
 
+def set_master_schedulable(facts):
```

Comment:
@detiber That sounds great to me. Do you know where I can find an example of a node fact? I've searched around various roles but I don't know what I'm looking for. Thanks for all the help.